DnsTube.Core/Services/IpAddressService.cs

```diff
@@ -93,7 +93,7 @@ public bool IsValidIpAddress(IpSupport protocol, string ipString)
 				if (splitValues.Length != 4)
 					return false;
 
-				return splitValues.All(r => byte.TryParse(r, out byte tempForParsing));
+				return splitValues.TrueForAll(r => byte.TryParse(r, out byte tempForParsing));
```

Comment:
Does this compile for you? It appears to be a static method, at least in .NET 7.

Reply:
removed